shuup/admin/settings.py

```diff
@@ -21,3 +21,12 @@
 #: Panes must be subclasses of `shuup.admin.views.WizardPane`.
 #:
 SHUUP_SETUP_WIZARD_PANE_SPEC = []
+
+#: Whether the admin checks per object permissions for the views and lists
+#:
+SHUUP_CHECK_PER_OBJECT_PERMISSIONS = False
```

Comment:
But to make multivendor work you need to set this True. I understood that we want to enable the addon without need for "Django-settings" changes. Well with this it's not possible.

Like said I would like that all of the multivendor-related stuff was under the same switch.